### ISSUE TITLE
Long size text overlap the check-marks from "Site URL/Is this information correct?" fields

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2218]
URL: www.staging.webcompat.com  
Browser / Version: Firefox Nightly 60.0a1 (2018-03-06) (Mobile & Desktop),  Chrome 64.0.3282.186 (Desktop)
Operating System: Windows 10 Pro, Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Steps to Reproduce:**
    1. Navigate to www.staging.webcompat.com.
    2. Tap “Report bug” button.
    3. Type in a long address in the “Site URL” field.
    4. Observe check-mark.
       
**Expected Behavior:**
No elements are overlapped.

**Actual Behavior:**
Text overlaps the check-mark.

**Note**
    1. Not reproducible on production (www.webcompat.com).
    2. Also reproducible for “Is this information correct?” fields.
    3. Affected area:
```html
<input class="form-field text-field required" id="url" name="url" pattern="" placeholder="e.g. https://example.com" required="" value="" type="url">
<span class="form-input-validation check error"></span>
```
        
**Watchers:**
@softvision-sergiulogigan
@softvision-oana-arbuzov

sv;

**Comment by softvision-oana-arbuzov** [comment]
Suggestion: adding `padding-right: 25px;`, no elements are overlapped.
```css
.text-field {
    display: block;
    width: 100%;
    padding: 4px 0;
        padding-right: 0px;
    border: none;
        border-bottom-width: medium;
        border-bottom-style: none;
        border-bottom-color: currentcolor;
    border-bottom: 1px solid rgba(0, 0, 0, .12);
    position: relative;
    z-index: 2;
}
```
![screenshot_2](https://user-images.githubusercontent.com/12184325/37093304-936029ec-2217-11e8-9f4d-e862eed0d2fe.png)

**Comment by deeptibaghel** [comment]
@zoepage can i take this up please?

**Comment by zoepage** [comment]
@deepthivenkat Yes! :)
Please remember to branch from refactor as this is a refactor task.

Let me know, if you have questions :)

**Comment by zoepage** [comment]
merged #2229
